docs/hub/models-widgets.md

```diff
@@ -67,12 +67,72 @@ them as:
 
 ```yaml
 widget:
-- src: https://huggingface.co/username/model_repo/resolve/main/sample1.flac
-  example_title: Custom Speech Sample 1
+  - src: https://huggingface.co/username/model_repo/resolve/main/sample1.flac
+    example_title: Custom Speech Sample 1
+```
+
+But even more convenient, if the file lives in the corresponding model repo, you can just use the filename or file path inside the repo:
+
+```yaml
+widget:
+  - src: sample1.flac
+    example_title: Custom Speech Sample 1
+```
+
+or if it was nested inside the repo:
+
+```yaml
+widget:
+  - src: nested/directory/sample1.flac
 ```
 
 We provide example inputs for some languages and most widget types in [the DefaultWidget.ts file](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/DefaultWidget.ts). If some examples are missing, we welcome PRs from the community to add them!
 
+## Example outputs
+
+As an extension to example inputs, for each widget example, you can also optionally describe the corresponding model output, direcly in the `output` property.
+
+This is useful when the model is not yet supported by the Inference API (for instance, the model library is not yet supported, or the model is too large), so that the model page can still showcase how the model works and what results it gives.
```

Comment:
```suggestion
This is useful when the model is not yet supported by the Inference API (for instance, the model library is not yet supported or the model is too large) so that the model page can still showcase how the model works and what results it gives.

```